### ISSUE TITLE
Site Assembler V2: Ensure the font and color variations are compatible with wireframe patterns

### ISSUE THREAD

**Issue by arthur791004** [Automattic/wp-calypso#71985]
#### Description

We have to ensure the wireframe patterns still look great after the user select any font and color variations

#### Related

* https://github.com/Automattic/wp-calypso/issues/71038

**Comment by autumnfjeld** [comment]
@alaczek  @arthur791004  This is issue is now extremely relevant given the problems with link colors that were surface during the walk through on March 22.  Even if we don't fix all the issues before shipping to users, we should make sure we can report that we know what the problem is and fixes are in the works.

**Comment by alaczek** [comment]
Yep, I'm on it :) 

So far, I checked all the navigation blocks in the headers and found no custom styling being applied. However, the default theme style is to use the same color for navigation links and regular text (with no underline), which is a popular choice in practice but might give the impression that nothing changes on the color switch. 

For the regular links in the content, the styling is minimal by default too. The color is the same as regular text, and they have an underline; on hover, the color changes to primary, and the underline goes away. I found a few instances where the link color was set deliberately in the pattern, and in those cases, the color did not change on hover. I'll think about whether to remove that custom link styling or not.
